src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPLIT_REGEX = ",";
+    private static final int TOTAL_BUY_INDEX = 0;
+    private static final int TOTAL_SUPPLY_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] strings = readFile(fromFileName);
+        int[] totalBuyAndSupply = generateReport(strings);
+        writeFile(toFileName, totalBuyAndSupply);
+    }
+
+    private int[] generateReport(String[] strings) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        for (String string : strings) {
+            String[] splitString = string.split(SPLIT_REGEX);
+            if (splitString[0].equals("supply")) {
+                totalSupply += Integer.parseInt(splitString[1]);
+            }
+            if (splitString[0].equals("buy")) {
+                totalBuy += Integer.parseInt(splitString[1]);
+            }
+        }
+
+        return new int[]{totalBuy, totalSupply};
```

Comment:
return a report here, your method is called generateReport, so return that report